### PR TITLE
[14.0] [MIG] partner_deduplicate_acl: Migration to v14.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT30291

please @joao-p-marques @pedrobaeza review it

### CODE REVIEW

File: partner_deduplicate_acl/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Ohne Beschränkungen"
+
+#~ msgid "Contact"
+#~ msgstr "Kontakt"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/de.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Sin restricciones"
+
+#~ msgid "Contact"
+#~ msgstr "Contacto"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/es.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/hr.po [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+msgid "Tools"
+msgstr "Alati"
+
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Bez ograničenja"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/hr.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/nl_NL.po [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+msgid "Tools"
+msgstr ""
+
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Zonder beperkingen"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/nl_NL.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/pt_BR.po [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+msgid "Tools"
+msgstr "Ferramentas"
+
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Sem restrições"
+
+#~ msgid "Contact"
+#~ msgstr "Contato"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/pt_BR.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/sk.po [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+msgid "Tools"
+msgstr "Nástroje"
+
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Bez obmedzení"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/sk.po]:
```suggestion
```

File: partner_deduplicate_acl/i18n/tr.po [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+msgid "Tools"
+msgstr "Araçlar"
+
+#. module: partner_deduplicate_acl
+#: model:res.groups,name:partner_deduplicate_acl.group_unrestricted
+msgid "Without restrictions"
+msgstr "Kısıtlama olmaksızın"
+
+#~ msgid "base.partner.merge.automatic.wizard"
+#~ msgstr "base.partner.merge.automatic.wizard"
```
Comment (victoralmau) [inline partner_deduplicate_acl/i18n/tr.po]:
```suggestion
```

File: partner_deduplicate_acl/tests/test_partner_deduplicate_acl.py [hunk 8]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2017-2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import exceptions
+from odoo.tests import common
+
+
+class TestPartnerDeduplicateAcl(common.TransactionCase):
+    def setUp(self):
+        super(TestPartnerDeduplicateAcl, self).setUp()
```
Comment (victoralmau) [inline partner_deduplicate_acl/tests/test_partner_deduplicate_acl.py]:
```suggestion
        super().setUp()
```

File: partner_deduplicate_acl/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Deduplicate Contacts ACL",
+    "summary": "Contact deduplication with fine-grained permission control",
+    "version": "14.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline partner_deduplicate_acl/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Some little changes. 
Maybe you have forgotten "_black, isort, prettier_" commit right?

**Comment by CarlosRoca13** [comment]
I tried to do it, but no changes were staged.

Other changes done

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**